cmd/edit/machinepool/cmd_test.go

```diff
@@ -81,8 +81,9 @@ var _ = Describe("Edit Machinepool", func() {
 				cmd := NewEditMachinePoolCommand()
 				Expect(cmd.Flag("cluster").Value.Set(clusterId)).To(Succeed())
 				Expect(cmd.Flags().Set("labels", "test=test")).To(Succeed())
+				Expect(cmd.Flags().Set("replicas", "1")).To(Succeed())
 				Expect(runner(context.Background(), t.RosaRuntime, cmd, []string{"--machinepool",
-					nodePoolId, "--min-replicas", "2", "--enable-autoscaling", "true", "-y"})).To(Succeed())
+					nodePoolId, "--replicas", "2", "--enable-autoscaling", "true", "-y"})).To(Succeed())
```

Comment:
Should this succeed? If you enable autoscaling you need to supply min/max instead of replicas, no?

Reply:
The args do not matter in testing, but i will remove them anyway (the new runner / cobra works differently when testing)